runtime/compiler/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -149,6 +150,67 @@ static const char* getNameForMethodRelocation (int type)
    return NULL;
    }
 
+uint8_t *
+J9::AheadOfTimeCompile::initializeCommonAOTRelocationHeader(TR::IteratedExternalRelocation *relocation, TR_RelocationRecord *reloRecord)
+   {
+   uint8_t *cursor = relocation->getRelocationData();
+
+   TR::Compilation *comp = TR::comp();
+   TR_RelocationRuntime *reloRuntime = comp->reloRuntime();
+   TR_RelocationTarget *reloTarget = reloRuntime->reloTarget();
+   TR::SymbolValidationManager *symValManager = comp->getSymbolValidationManager();
+   TR_J9VMBase *fej9 = comp->fej9();
+   TR_SharedCache *sharedCache = fej9->sharedCache();
+
+   TR_ExternalRelocationTargetKind kind = relocation->getTargetKind();
+
+   switch (kind)
```

Comment:
Maybe a simple comment is warranted for these two `switch` statements so that any casual reader doesn't come along and see a clean-up opportunity :) .